pkg/rtc/participant.go

```diff
@@ -182,7 +184,9 @@ func NewParticipant(params ParticipantParams) (*ParticipantImpl, error) {
 	})
 
 	primaryPC := p.publisher.pc
-
+	// primary connection does not change, canSubscribe can change if permission was updated
+	// after the participant has joined
+	p.subscriberAsPrimary = p.ProtocolVersion().SubscriberAsPrimary() && p.CanSubscribe()
 	if p.SubscriberAsPrimary() {
```

Comment:
What happens when subscription is allowed later? We are not creating data channels on the subscriber PC or are we doing it elsewhere when permission changes?

I am guessing there will be more issue if we create later with migration mid ordering and stuff.

Reply:
Thank you